### ISSUE TITLE
Cannot group with more than one field name

### ISSUE THREAD

**Issue by prusfr** [VianneyMI/monggregate#75]
Pipeline.group() method wont' allow more than 1 field name of class string, whereas doc indicates set or list can be passed.
Example :
`Pipeline().group(by=["field1", "field2"])` will throw an error.
Thus, multi-fields group cannot be set and such a syntax : `{$group: {$_id : { field1: "$field1", field2: "$field2"}}}` cannot be achived

**Comment by VianneyMI** [comment]
Thanks for the report.

Yes the doc is a bit misleading. I am currently working on creating a proper documentation for the package. 
I'll fix this soon. 

In the meantime you can still write:

```python
Pipeline().group(    
        query = {
            "field1":"$field1",
            "field2":"$field2",
        }
    )
```

**Comment by VianneyMI** [comment]
Fixed in 0.16.2
